src/function_classes/decision_tree.py

```diff
@@ -32,7 +32,7 @@ def _init_param_dist(self) -> D.Distribution:
         )
 
     def evaluate(self, x_batch: torch.Tensor, *params: torch.Tensor) -> torch.Tensor:
-        dt_tensor, target_tensor, _ = params
+        dt_tensor, target_tensor, *_ = params
```

Comment:
this fix is correct here because `params` is of length 2, and the elements are correctly the `dt_tensor` and `target_tensor`